### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Drake713** [LennardF1989/NiohResolution#4]
After getting to install successfully the game kept crashing upon killing a Yokai demon - each time the game would crash just as I stuck the killing blow.

I restored the .bak (changed nothing else) and it stopped crashing there

**Comment by LennardF1989** [comment]
Is it a specific demon, or any demon?

**Comment by Drake713** [comment]
I was only killing the one so can't say for sure if it is just the one or any of them - for me it was the Yokai just before the hidden healing spring in the Umi-Bozu mission and not the one behind a curtain, it was the one standing in the middle of the hallway

**Comment by LennardF1989** [comment]
I'll leave this one open for others to verify, thanks! Given this is a reversed engineered patch, I'm afraid there isn't really much there can been done about it, other than allowing opt-out of the aspect ratio fix.

**Comment by Drake713** [comment]
If it helps I was using the following settings:
1680
1050
1

**Comment by msoltyspl** [comment]
While superbly rarely, I actually experienced that as well running at 1680x1050. [This](https://i.imgur.com/f9iBQ08.jpg) yokai (beginning of L90 twilight mission "The bleeding spider Lily") is basically a guaranteed crash on kill with modified exe. 0 crashes in standard borderless (1600x900)

**Comment by LennardF1989** [comment]
Very weird, but thanks for reporting! I might look into a proxy hack rather than an assembly hack soon.

**Comment by GreenAmaranthine** [comment]
I can corroborate this as well. 1920x1200, not that I suspect it matters. For me it was crashing every single time I killed ANY enemy with anything but my fists. This is a well-known bug with the PC version of the game in general related to dismemberment, and nobody seems to know what actually causes it or how to fix it (many different solutions work for many different people), and the problem would persist for me even when using a standard resolution (1280x720) in windowed mode, until I restored the backup exe. That happened to be what fixed it for me.

Weird thing is the game worked fine with the patch for 36 hours prior to that. Tried saving after the problem was fixed and reapplying the patch and it immediately started crashing again.
